_squirtle/bash-hook

```diff
@@ -4,6 +4,10 @@ function _sq_set_secret() {
     then
         export SOROBAN_SECRET_KEY=$(</workspace/.soroban-secret-key)
     fi
+    if [[ -f /workspace/.soroban-rpc-url ]]
+    then
+        export SOROBAN_RPC_URL=$(</workspace/.soroban-rpc-url)
+    fi
 }
 
-trap '_sq_set_secret' DEBUG
\ No newline at end of file
+trap '_sq_set_secret' DEBUG
```

Comment:
Do we need this line? What does it do?

Reply:
This line existed before, it sets our function as a "DEBUG trap" this is normally meant to debug bash scripts, as the trap is run before each executed command, but for our case it allows us to modify the env before running user commands.